README.md

```diff
@@ -76,11 +67,23 @@ A valid (read: that follows this interface) IPLD format implementation the follo
 
 `callback` must have the signature `function (err, dagNode)`, where `err` is an Error if the function fails and `dagNode` is the dagNode that got deserialized in the process.
 
-#### `util.cid(dagNode, callback)`
+#### `util.cid(binaryBlob[, options], callback)`
+
+> get the CID of the serialized dagNode
```

Comment:
Note: in some formats, figuring out the correct hash algorithm to use may require deserializing the node again. Probably not much of an issue but still, FYI.

Reply:
Do you have an example?

I see that sometimes you can't even figure it out, you'd need to pass in as option. For example you have CBOR with a non-default Hash.

Reply:
> Do you have an example?

No, I'm just playing the devil's advocate.

> I see that sometimes you can't even figure it out, you'd need to pass in as option. For example you have CBOR with a non-default Hash.

CBOR should default to sha2-256, IIRC.